### DISCUSSION TITLE
How to override values of a config group in a new config group

### DISCUSSION THREAD

**Discussion by DirkKuhn** [mit-ll-responsible-ai/hydra-zen#476] [Q&A]
Hi,

what is the recommended way to override values of a config group with a new config group?
```python
class Item:
    def __init__(self, msg: str):
        self.msg = msg

item_store = store(group='item')
item_store(Item, msg='default', name='default')

item_store(
    make_config(
        hydra_defaults=['default', '_self_'],
        msg='overriden',
        bases=(store['item', 'default'],)
    ),
    name='override'
)
```
The config group with name 'override' just uses the config group with name 'default' but changes the value of msg.
In practise this might be usefull to avoid importing `Item`.
Is there a better way or should this be avoided entirely?

Thanks so much!

**Comment by jgbos** [comment]
Hi @DirkKuhn, did you find a good way to do this?  Did you take a look at the experiment configuration approach documented here: https://mit-ll-responsible-ai.github.io/hydra-zen/how_to/configuring_experiments.html?  That is a really nice way to configure different values for an experiment.  Not sure that's what you are looking for though.

**Reply by DirkKuhn** [reply]
Thank you for your answer. In hindsight I realized that my question does not make a lot of sense.
I am no longer using something like this. I will delete this question now.
